packages/calcite-components/src/tests/commonTests/index.ts

```diff
@@ -0,0 +1,26 @@
+export { accessible } from "./accessible";
+export { defaults } from "./defaults";
+export { openClose } from "./openClose";
+export { reflects } from "./reflects";
+export { renders } from "./renders";
+export { disabled } from "./disabled";
+export { hidden } from "./hidden";
+export { floatingUIOwner, delegatesToFloatingUiOwningComponent } from "./floatingUI";
+export { focusable } from "./focusable";
+export { formAssociated } from "./formAssociated";
+export { slots } from "./slots";
+export { labelable } from "./labelable";
+export { t9n } from "./t9n";
+export {
+  TagAndPage,
+  TagOrHTMLWithBeforeContent,
+  isHTML,
+  getTag,
+  simplePageSetup,
+  getTagAndPage,
+  ComponentTestSetupProvider,
+  ComponentTestContent,
+  ComponentTestSetup,
+  getTagOrHTMLWithBeforeContent,
+  HYDRATED_ATTR,
```

Comment:
Except for `HYDRATED_ATTR`, the rest should not be exported as they are internal to the common test modules.

Sidebar: can you create a follow-up issue to update/remove tests using it in favor of `renders`?

Reply:
https://github.com/Esri/calcite-design-system/issues/9166